models/nulog/NuLogParser.py

```diff
@@ -267,7 +267,7 @@ def get_test_dataloaders(self, data_tokenized, transform_to_tensor):
     def load_data(self, windows_folder_path):
         try:
             df_log = self.log_to_dataframe(windows_folder_path)
-            return [df_log.iloc[i].Content for i in range(df_log.shape[0])]
+            return list(df_log[~df_log["Content"].isnull()]["Content"])
```

Comment:
pls change to `df_log[~df_log["Content"].isnull()]["Content"].tolist()`

Reply:
done.